readers/messages.go

```diff
@@ -36,6 +36,9 @@ type MessageRepository interface {
 
 	// Backup retrieves all messages from database.
 	Backup(rpm PageMetadata) (MessagesPage, error)
+
+	// Deletes messages for a specific publisher within a time range. Returns number of deleted messages.
```

Comment:
Are you sure that we return the number of deleted messages? Looks like we only return an error.